skbio/util/tests/test_misc.py

```diff
@@ -231,5 +233,41 @@ def gen():
         self.assertEqual(find_duplicates(gen()), set(['a', 2]))
 
 
+class TestGetRng(unittest.TestCase):
```

Comment:
We should probably add a test to make sure that the seeds are disjoint.

Maybe for one of the use cases (i.e. subsample), we should have a test generating two random generators with 2 different seeds and assert that the subsampled counts aren't identical.